### PR TITLE
Add Galileo and Beidou SSR support

### PR DESCRIPTION
**Pull request by scarcanague**

### CODE REVIEW

File: c/src/rtcm3_ssr_decode.c [hunk 1]
```diff
@@ -10,29 +10,101 @@
...
+    case CONSTELLATION_BDS2:
+    case CONSTELLATION_QZS:
+    case CONSTELLATION_SBAS:
+      return 20;
+    case CONSTELLATION_GLO:
+      return 17;
+    case CONSTELLATION_INVALID:
+    case CONSTELLATION_COUNT:
+    default:
+      assert(false);
```
Comment (anth-cole) [inline c/src/rtcm3_ssr_decode.c]: I wouldn't assert in here as it'll bring down whatever is running the convertor, I'd just return a message invalid - we don't want to crash if we get a corrupted message somehow

File: c/src/rtcm3_ssr_decode.c [hunk 2]
```diff
@@ -10,29 +10,101 @@
...
+      return 0;
+  }
+}
+
+/*
+*            TYPE       GPS     GLOASS    GALILEO    QZSS     BEIDOU     SBAS
+*         ----------------------------------------------------------------------
+*
+*          SSR OBT   : 1057      1063      1240*     1246*     1258*       -
```
Comment (anth-cole) [inline c/src/rtcm3_ssr_decode.c]: Can you put a comment explaining the `*`?

Comment (scarcanague) [reply c/src/rtcm3_ssr_decode.c]: Done (it is just indicate these RTCM message are still draft)

File: c/src/rtcm3_ssr_decode.c (hunk 1 above)
Comment (scarcanague) [reply c/src/rtcm3_ssr_decode.c]: Done!

**Pull request merged**